publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -272,15 +273,36 @@ public void createRabbitMqConnection() throws RemRemPublishException {
             }
             factory.setConnectionTimeout(tcpTimeOut);
             rabbitConnection = factory.newConnection();
+            rabbitConnection.addShutdownListener(new ShutdownListener() {
+                @Override
+                public void shutdownCompleted(ShutdownSignalException cause) {
+                    log.debug("Connection Shutdown completed " + cause);
+                    try {
```

Comment:
I think the section is useless. The connection is always closed at that point of time...

Reply:
I have tested by removing this piece of code than I'm seeing more connections..,

Reply:
Hmm, very interesting... In that case we should change throwable to IOException or AlreadyClosedException or something what is thrown. Next, e.printStackTrace() should be removed---in fact we don't need any code there. Instead add a comment explainig why we have this try/catch block and that should not be removed.

Reply:
Sure Roman